disconnect-when-backgrounded.js

```diff
@@ -16,7 +17,9 @@ function disconnectIfHidden() {
     removeDisconnectTimeout();
 
     if (document.hidden) {
-        createDisconnectTimeout();
+        if((Package["iron:router"] && disconnectVoids.indexOf(Router.current().route.getName()) < 0) || (!Package["iron:router"])){
```

Comment:
should be: `!Package["iron:router"] || disconnectVoids.indexOf(Router.current().route.getName()) < 0`